### PR TITLE
Update add-custom-page.md

### PR DESCRIPTION
**Pull request by jrson83**
- fix broken anchor links
- add additional info regarding necessary adjustments of the page-loader to the `services.xml`, which are needed to follow the guide and not get [stuck](https://forum.shopware.com/t/add-custom-page-dokumentation-too-few-arguments/88072)

### CODE REVIEW

File: guides/plugins/plugins/storefront/add-custom-page.md [hunk 1]
```diff
@@ -162,6 +162,23 @@ class ExampleController extends StorefrontController
 
 Note, that we've added the page to the template variables.
 
+#### Adjusting the services.xml
+
+In addition, it is necessary to pass the argument with the ID of the `ExamplePageLoader` class to the [configuration ](add-custom-controller#services-xml-example) of the controller service in the `services.xml`.
+
+```html
```
Comment (mitelg) [inline guides/plugins/plugins/storefront/add-custom-page.md:169]: shouldn't that be 

```suggestion
```xml
```

Comment (jrson83) [reply guides/plugins/plugins/storefront/add-custom-page.md:169]: I was wondering why on some pages code blocks for xml files you are using `xml` and on some `html`.
As I compared the pages I was thinking it is consistent using `html` for xml files not including the full xml markup.

Can you confirm this? Else there are many pages with wrong code block tags using html for xml files.

Comment (mitelg) [reply guides/plugins/plugins/storefront/add-custom-page.md:169]: If you want to fix it for all occurrences, I would appreciate it, but if not, no worries, we can keep it like that for consistency

Comment (jrson83) [reply guides/plugins/plugins/storefront/add-custom-page.md:169]: Lets say we keep it for this PR and I will check on how many blocks the tag is wrong and see then later today?

Comment (mitelg) [reply guides/plugins/plugins/storefront/add-custom-page.md:169]: totally fine for me :ok_hand:

**Review by mitelg — APPROVED** [review]

**Comment by mitelg** [comment]
thanks for your contribution @jrson83 👍🎉💙

**Pull request merged**